concrete-csprng/Cargo.toml

```diff
@@ -20,13 +20,17 @@ rand = "0.8.3"
 criterion = "0.3"
 
 [features]
-slow = []
-multithread = ["rayon"]
+parallel = ["rayon"]
+seeder_rdseed = []
+seeder_linux = []
+generator_aesni = []
+generator_soft = []
 
 [[bench]]
 name = "benchmark"
 harness = false
 
 [[bin]]
```

Comment:
Why do we have a main.rs actually?

Reply:
This generates a program that outputs random bytes on stdout. We used this in the past to verify the prng was cryptographically secure, using some statistical analysis tool.

Reply:
Should we do that for the linux generator then?

Reply:
Not really I think. Unless we want to put that on CI or something.